src/chart/pie/pieLayout.ts

```diff
@@ -113,12 +116,19 @@ export default function pieLayout(
         const extent = data.getDataExtent(valueDim);
         extent[0] = 0;
 
+        const dir = clockwise ? 1 : -1;
+        const angles = [startAngle, endAngle];
+        normalizeArcAngles(angles, !clockwise);
+
+        [startAngle, endAngle] = angles;
+
+        const angleRange = Math.abs(endAngle - startAngle);
```

Comment:
I don't think we need `Math.abs` here since `normalizeArcAngles` has done the job to normalize them to make sure `endAngle` is larger. And this will look the same as [GaugeView](https://github.com/apache/echarts/blob/master/src/chart/gauge/GaugeView.ts#L122-L126).

Reply:
From my local test, normalizeArcAngles not make endAgnle larger when anticlockwise

<img width="1798" alt="image" src="https://github.com/apache/echarts/assets/56526981/a80eadfd-545b-490c-9e67-578dfc2a467a">

and in `normalizeArcAngles` comment, that has should be negative when anticlockwise, but now is positive, so I have to add abs here, to avoid get a negative range.

<img width="644" alt="image" src="https://github.com/apache/echarts/assets/56526981/def20657-8706-4585-b802-0f50d31abdde">


If there is a zrender bug here, may be we should add abs to here for now. and after this 'bug'(maybe) fixed, then remove this.

Reply:
Thanks for correction. I think you are right. I've tested and it works as expected.